### PR TITLE
Create custom html carousel template

### PR DESCRIPTION
**Pull request by Koji23**
https://user-images.githubusercontent.com/10385763/177383328-3d20ecae-6c0b-4c7c-b305-364acf07f7a3.mov

### CODE REVIEW

File: carousel/index.html [hunk 1]
```diff
@@ -0,0 +1,976 @@
+<html>
+  <head>
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no, viewport-fit=cover"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
```
Comment (gardenFiend138) [inline carousel/index.html:7]: is this duplicated in the lines below?

File: carousel/index.html [hunk 2]
```diff
@@ -0,0 +1,976 @@
...
+
+        document
+          .querySelector(".get-started")
+          .addEventListener("click", function (e) {
+            OneSignalIamApi.close(e);
+          });
+
+        document.querySelectorAll(".go-to-next-page").forEach(function (node) {
+          node.addEventListener("click", function (e) {
+            console.log(iamInfo.carousel.currentPage);
```
Comment (gardenFiend138) [inline carousel/index.html:967]: do we intend for this to be logged?

File: carousel/index.html [hunk 3]
```diff
@@ -0,0 +1,976 @@
...
+              }
+
+              handleEndOfDrag();
+            }
+          },
+          false
+        );
+      });
+
+      // NOTE: for "resize" and "orientationchange" events
```
Comment (gardenFiend138) [inline carousel/index.html:889]: i like that this comment explains why we're doing the multiple calls to setupCarouselPages in timeouts, but i'm wondering how relevant this is for our intended audience (non-technical folks)? 

last sentence in this comment needs revising:
> we are opting to make multiple calls function calls.

**Review by gardenFiend138 — COMMENTED** [review]
looks pretty slick! can we add Olive and Charles to this review as well? And have we tested across device types/ screen sizes/ landscape mode?

File: carousel/index.html (hunk 1 above)
Comment (Koji23) [reply carousel/index.html:7]: woops yes it is, thanks for that catch

File: carousel/index.html (hunk 2 above)
Comment (Koji23) [reply carousel/index.html:967]: ah no should be removed

**Comment by Koji23** [comment]
@gardenFiend138 I believe we will need to add a new method to the iam-api that disables vertical-drag-to-dismiss-iam. This is a useful default feature for most iams but for certain ones like carousel that have specific drag based UX we would want to disable it to prevent accidental dismissals

**Review by gardenFiend138 — COMMENTED** [review]
looks like maybe your changes never got pushed?

**Comment by Koji23** [comment]
> looks like maybe your changes never got pushed?

Woops yes must've missed it somehow, just pushed

**Review by iAmWillShepherd — APPROVED** [review]
Token approval

**Pull request merged**